contracts/managers/price-triggers/RSITrendingTrigger.sol

```diff
@@ -71,36 +74,43 @@ contract RSITrendingTrigger is
             "RSITrendingTrigger.constructor: Upper bound must be greater than lower bound"
```

Comment:
Just noticed this. To generalize it more, should be `_upperBound >= _lowerBound`, in case a single RSI value acts as a trigger